Gemfile

```diff
@@ -36,6 +36,11 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# gems for api consumption
+gem 'faraday'
+gem 'jsonapi-serializer'
+gem 'json' 
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
```

Comment:
Did we decide on Travis CI afterall, should we add gem 'travis' ?

Reply:
I thought we did but we can clarify in the stand-up today.